Documentation/Architecture/InputSystem/CoreSystem.md

```diff
@@ -8,11 +8,8 @@ At the heart of the input system is the [InputSystem](../../Input/Overview.md),
 
 This service is responsible for:
 
-- Reading the [input system profile](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityInputSystemProfile.cs)
-- Starting the various device managers (for example, [OpenVR](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.Providers/OpenVR/OpenVRDeviceManager.cs),
-  [Windows Mixed Reality](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityDeviceManager.cs),
-  [Unity Touch](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit/Providers/UnityInput/UnityTouchDeviceManager.cs)).
-  The set of device managers that are instantiated is configured by the input system profile.
+- Reading the [input system profile](../../MixedRealityConfigurationGuide.md#input-system-settings)
+- Starting the configured [data providers](../../Input/InputProviders.md) (for example, `Windows Mixed Reality Device Manager` and `OpenVR Device Manager`).
```

Comment:
add WindowsMixedRealityDeviceManager and other xref?

Reply:
there are currently no api doc pages for these classes.

Reply:
Issue #7534 tracks the missing doc pages